### PR TITLE
Update Admin Settings User Input Design

### PR DESCRIPTION
**Pull request by nfmohit**
## Summary

Addresses issue:

- https://github.com/google/site-kit-wp/issues/5896#issuecomment-1378776259

## Relevant technical choices

This PR addresses the concerns raised by QA [here](https://github.com/google/site-kit-wp/issues/5896#issuecomment-1378776259). Summary:

1. Aligns user input options in Preview screen to the left according to Figma (addressing concern 2).
2. Updates text color and typography of questions in Preview screen (addressing concern 3).
3. When the user input settings haven't loaded yet, the Preview screen shows an error. This PR shows a loader until the settings have loaded (addressing concern 4).
4. Removes separation between user input options in Preview screen (addressing concern 5).

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/sass/config/_variables.scss [hunk 1]
```diff
@@ -68,7 +68,7 @@ $c-gallery: #ebeef0;
 $c-gallery-2: #ebeef0;
 $c-fiord: #1b3870;
 $c-iron: #b8bdb9;
-$c-shark: #232824;
+$c-heavy-metal: #232824;
```
Comment (nfmohit) [inline assets/sass/config/_variables.scss:71]: This variable is renamed according to http://chir.ag/projects/name-that-color/. Regardless, this is unused in the codebase at the moment.

File: assets/sass/components/user-input/googlesitekit-user-input-preview.scss [hunk 2]
```diff
@@ -144,4 +144,12 @@
 			}
 		}
 	}
+
+	.googlesitekit-user-input__preview-group .googlesitekit-user-input__select-option .mdc-radio {
```
Comment (asvinb) [inline assets/sass/components/user-input/googlesitekit-user-input-preview.scss:148]: @nfmohit Let's simplify those lines into the following:
```css
.googlesitekit-user-input__preview-group {
    .googlesitekit-user-input__select-option {
        .mdc-radio {
            margin-left: -10px;
        }

        &:not(:last-child) {
            border-bottom: none;
        }
    }
}
```

File: assets/sass/config/_variables.scss (hunk 1 above)
Comment (asvinb) [reply assets/sass/config/_variables.scss:71]: @nfmohit I would revert the changes in this file and instead add the new color in 
`assets/sass/config/_variables-mui3.scss`
image

We can name it `$c-grey-g-900`?

**Review by asvinb — CHANGES_REQUESTED** [review]
@nfmohit Just left a few minor changes. Let me know what you think :)

File: assets/sass/components/user-input/googlesitekit-user-input-preview.scss (hunk 2 above)
Comment (nfmohit) [reply assets/sass/components/user-input/googlesitekit-user-input-preview.scss]: Updated, thank you @asvinb!

File: assets/sass/config/_variables.scss (hunk 1 above)
Comment (nfmohit) [reply assets/sass/config/_variables.scss]: Updated, thank you @asvinb!

**Comment by nfmohit** [comment]
> @nfmohit Just left a few minor changes. Let me know what you think :)

Addressed the changes, thank you @asvinb!

**Comment by asvinb** [comment]
LGTM @nfmohit !

**Review by eugene-manuilov — APPROVED** [review]
LGTM

**Pull request merged**